charts/aws-eks-asg-rolling-update-handler/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: aws-eks-asg-rolling-update-handler
-version: 1.3.0
+version: 1.3.1
```

```suggestion
version: 1.4.0
```
Since this is a new feature, we should bump the major version